lib/utils.js

```diff
@@ -0,0 +1,115 @@
+function getConfigurations(isCachingEnabled) {
+    // TODO: implement a proper caching mechanism
+    var configs = require("configs.json");
+    configs.isCachingEnabled = isCachingEnabled;
+    return configs;
+}
+
+function getLookUpTable(configs) {
+    // TODO: implement a proper caching mechanism
+    var log = new Log("utils-app-context");
+
+    function getLayoutsData(layoutsDir) {
```

Comment:
Fixed in PR https://github.com/wso2/carbon-uuf/pull/2 (see commit https://github.com/sajithar/carbon-uuf/commit/a6209253ee164e5c10712f7b14a4dcfd665adb41)